plugin.php

```diff
@@ -130,7 +130,7 @@
 add_filter( 'init', '_add_extra_api_post_type_arguments', 11 );
 add_action( 'init', '_add_extra_api_taxonomy_arguments', 11 );
 add_action( 'rest_api_init', 'rest_register_settings', 0 );
```

Comment:
I do not quite get it, why `0` was used as priority. The `rest_register_setting()` only registers like `posts_per_page` but does not invoke critical REST API components like `create_inital_rest_routes()` does. So to me, it looks save, but if there is no reason for priority `0`, I think we should switch to default? /cc @joehoyle